src/jquery.ime.inputmethods.js

```diff
@@ -198,6 +198,10 @@
 			name: 'फोनेटिक',
 			source: 'rules/hi/hi-phonetic.js'
 		},
+		'hoc': {
+			name: '𑢹𑣉𑣉/Ho',
+			source: 'rules/hoc/hoc-transliteration.js'
+		},
 		'is-normforms': {
```

Comment:
One more entry required to  map this input method to language in this same file. Please refer other languages. Without that the IME wont work. Please make sure you test and confirm it is working